app/jobs/api_updater.rb

```diff
@@ -1,16 +1,12 @@
 class ApiUpdater < Updater
-  def perform(recurring = true)
-    begin
-      import
-      pre_grouper = PreGrouper.new
-      pre_grouper.perform
-      grouper = Grouper.new
-      grouper.perform(should_group_genes?, should_group_drugs?)
-      post_grouper = PostGrouper.new
-      post_grouper.perform(should_cleanup_gene_claims?)
-    ensure
-      reschedule if recurring
-    end
+  def perform()
```

Comment:
Don't actually have to change it, just a nitpicky FYI, but generally for zero argument methods, you'd omit the `()` but that's just a common idiom, not required/has no impact on anything.